src/vlogger/vlogger.h

```diff
@@ -259,6 +259,9 @@ static inline void vlog_printf(vlog_levels_t log_level, const char* fmt , ... )
 		len += snprintf(buf+len, VLOGGER_STR_SIZE-len-1, " %s %s: ", g_vlogger_module_name, log_level::to_str(log_level));
 	}
 
+	if (len < 0) {
+		return ;
```

Comment:
Rather that returning silently - it is better to print something sensible before returning.